### PR TITLE
Update /etc/cobbler/dnsmasq.template according to network_*.yaml and sync cobbler

### PR DESCRIPTION
**Pull request by kendriu**
Operation can be done only on master node

### CODE REVIEW

File: fuelclient/cli/actions/network.py [hunk 1]
```diff
@@ -12,10 +12,26 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import netaddr
```
Comment (prmtl) [inline fuelclient/cli/actions/network.py:15]: There is no netaddr in requirements.txt

File: fuelclient/cli/actions/network.py [hunk 2]
```diff
@@ -58,6 +79,38 @@ def upload(self, params):
...
+
+    @master_only
+    def _update_dnsmasq_template(self, networks):
+        fw_admin_nets = dict([(n['group_id'], n) for n in networks if
+                              n['name'] == 'fuelweb_admin'])
+        base_net = fw_admin_nets.pop(None)
+        for group_id, net in fw_admin_nets.iteritems():
+            marker = GROUP_ID_TEMPLATE.format(group_id = group_id)
+            try:
+                exec_cmd(COBBLER_SHELL + 'grep -q "%s" %s' % (
```
Comment (prmtl) [inline fuelclient/cli/actions/network.py]: I would make "exec_cmd_on_cobbler" or "exec_cmd_on_container" (depending how much you want to make it reusable)

File: fuelclient/cli/actions/network.py [hunk 3]
```diff
@@ -12,10 +12,26 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import netaddr
+from fuelclient.cli import error
 from fuelclient.cli.actions.base import Action
 import fuelclient.cli.arguments as Args
 from fuelclient.cli.arguments import group
 from fuelclient.objects.environment import Environment
+from fuelclient.objects.plugins import master_only
```
Comment (prmtl) [inline fuelclient/cli/actions/network.py]: `master_only` could be now move to some more general use file (utils? decorators?)

File: fuelclient/cli/actions/network.py (hunk 2 above)
Comment (prmtl) [inline fuelclient/cli/actions/network.py]: And for consistency, it could use .format()

**Comment by prmtl** [comment]
:shipit:

File: fuelclient/cli/actions/network.py (hunk 1 above)
Comment (kendriu) [inline fuelclient/cli/actions/network.py:15]: Done

File: fuelclient/cli/actions/network.py (hunk 2 above)
Comment (kendriu) [inline fuelclient/cli/actions/network.py]: done

**Pull request closed without merging**